clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -662,6 +662,7 @@ public void testInvalidTopicNames() throws Exception {
         }
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
@cmccabe I think we can keep most of those test as-is until `retries` is removed -- the ones with `@SuppressWarning` should be rewritten if `retries` is removed IMHO, while the ones with `@Deprecated` should be remove. Let me know if this makes sense or if you think we should rewrite those tests right away.

Reply:
Thanks @mjsax. I agree we should keep testing the deprecated option. It does kind of seem like we should rewrite the tests you want to rewrite, now, though. I don't think this scheme of deprecating some tests and suppressing the warning on others is going to be remembered for very long after this is merged.

Reply:
We can do that. Wanted to get input from @hachikuji and @cmccabe about it. Frankly, I am not always 100% sure atm how to exactly rewrite the tests and what test coverage is desired?

Reply:
I dug into the test and I think I found ways to get rid for `retries` for the methods that suppress the deprecation warning. Let me know what you think.